packages/app/obojobo-repository/shared/components/dashboard.test.js

```diff
@@ -1059,6 +1070,75 @@ describe('Dashboard', () => {
 		component.unmount()
 	})
 
+	test('updates selected module count properly when all modules are deselected', () => {
```

Comment:
The indices chosen here (`standardModules[0]`, `moduleComponents[4]`, etc.) are a bit magical - may be prudent to add a comment or two near those explaining why we're using those numbers in particular for the benefit of future devs.